Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "milagro_bls"
-version = "1.0.1"
+version = "1.0.0"
```

Comment:
Looks like we've gone backwards here?

Reply:
Yep I accidentally skipped `1.0.0` and we never made a release of `1.0.1`
Happy to change it to `1.0.1` if you think that's better?

Reply:
`1.0.0` is fine by me :)